source/install/install-mmte-helm-gitlab-helm.rst

```diff
@@ -0,0 +1,237 @@
+          exit 0
+          else
+          echo "Database does not exist. creating...."
+          PGPASSWORD=$POSTGRES_PASSWORD_GITLAB createdb -h $POSTGRES_HOST_GITLAB -p $POSTGRES_PORT_GITLAB -U $POSTGRES_USER_GITLAB $POSTGRES_DB_NAME_MATTERMOST
+          echo "Done"
+          fi
+    - name: create-minio-bucket
+      image: "minio/mc:RELEASE.2018-07-13T00-53-22Z"
+      env:
+        - name: MINIO_ENDPOINT
+          value: <GITLAB.MINIO.HOST>
+        - name: MINIO_PORT
+          value: "<GITLAB.MINIO.PORT>"
+        - name: MINIO_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: <GITLAB.MINIO.SECRET>
+              key: accesskey
+        - name: MINIO_SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: <GITLAB.MINIO.SECRET>
+              key: secretkey
+        - name: MATTERMOST_BUCKET_NAME
+          value: <MATTERMOST.MINIO.BUCKET.NAME>
+      command:
+        - sh
+        - "-c"
+        - |
+          echo "Connecting to Minio server: http://$MINIO_ENDPOINT:$MINIO_PORT"
+          mc config host add myminio http://$MINIO_ENDPOINT:$MINIO_PORT $MINIO_ACCESS_KEY $MINIO_SECRET_KEY
+          /usr/bin/mc ls myminio
+          echo $?
+          /usr/bin/mc ls myminio/$MATTERMOST_BUCKET_NAME > /dev/null 2>&1
+          if [ $? -eq 1 ] ; then
+            echo "Creating bucket '$MATTERMOST_BUCKET_NAME'"
+            /usr/bin/mc mb myminio/$MATTERMOST_BUCKET_NAME
+          else
+            echo "Bucket '$MATTERMOST_BUCKET_NAME' already exists."
+            exit 0
+          fi
+
+
+Values that you need to replace in the ``values.yaml``:
+
+- **<YOUR.MATTERMOST.DOMAIN>**: Your desired Mattermost domain. eg, ``mattermost.gitlab.example.com``
+- **<NAME.OF.YOUR.TLS.SECRET>**: A name to store the TLS certificate for you domains, eg. ``mattermost-tls``
+- **<INGRESS.CLASS>**: the ingress class, in a basic deployment of GitLab will be ``gitlab-nginx``
+- **<INGRESS.PROVIDER>**: the ingress provider, in a basic deployment of GitLab will be ``nginx``
+- **<CERTMANAGER.ISSUER>**: the cert manager issuer, in a basic deployment of GitLab will be ``gitlab-issuer``
+- **<GITLAB.APP.SECRET>**: The Application secret. The value you created in the step `Create the OAUTH with GitLab`_
+- **<GITLAB.APP.ID>**: The Application secret. The value you created in the step `Create the OAUTH with GitLab`_
+- **<YOUR.GITLAB.DOMAIN>**: The GitLab domain name. eg. ``gitlab.example.com``
+- **<GITLAB.POSTGRES.USERNAME>**: The GitLab Postgres username
+- **<GITLAB.POSTGRES.PASSWD.SECRET>**: Secret that holds the Postgres password
+- **<GITLAB.POSTGRES.HOST>**: Postgres host. Check the Kubernetes service, in a basic deployment of GitLab will be ``gitlab-postgresql``
+- **<GITLAB.POSTGRES.PORT>**: Postgres port. Check the Kubernetes service, in a basic deployment of GitLab will be ``5432``
+- **<MATTERMOST.DATABASE.NAME>**: Mattermost database name that you choose, eg. ``mattermost-db``
```

Comment:
Maybe just give a good default for this. No need to customize here, if someone wants to customize then they will know how.

Reply:
i will add the defaults here.